dev-environment/pmml-samples/.gitignore

```diff
@@ -0,0 +1,3 @@
+# Exports
```

Comment:
Unused

Reply:
This is where `.pmml` files are once you run training script.